sdk/src/wallet/operations/balance.rs

```diff
@@ -22,340 +22,345 @@ where
 {
     /// Get the balance of the wallet.
     pub async fn balance(&self) -> Result<Balance> {
+        #[cfg(not(target_family = "wasm"))]
+        {
+            let wallet = self.clone();
+            tokio::task::spawn_blocking(move || tokio::runtime::Handle::current().block_on(wallet.balance_inner()))
+                .await?
+        }
+        #[cfg(target_family = "wasm")]
+        {
+            self.balance_inner().await
+        }
+    }
+
+    async fn balance_inner(&self) -> Result<Balance> {
```

Comment:
This all isn't needed as it changes nothing? Not sure how we ended up like this, but looks like it works perfectly fine without any weird task spawning/blocking now
```suggestion
```

Reply:
the call to the client can